app-admin/ruri/spec

```diff
@@ -0,0 +1,4 @@
+VER=3.8
+SRCS="git::commit=tags/v$VER;copy-repo=true::https://github.com/Moe-Hacker/ruri"
+CHKSUMS="SKIP"
+CHKUPDATE="github::repo=Moe-Hacker/ruri"
```

Comment:
Please use anitya.

Reply:
Unable to retrieve the latest uersion from upstream!ERROR:"Error when checking for new version:Rate limit was reached.Will be reset in\"2024-12-31 T03:01:53+00:00\".

Reply:
> Unable to retrieve the latest uersion from upstream!ERROR:"Error when checking for new version:Rate limit was reached.Will be reset in"2024-12-31 T03:01:53+00:00".

Let them be. That ID would update tomorrow, they seem to be having traffic problems.

Reply:
OK.

Reply:
> OK.

So please change it?

Reply:
![Screenshot_2024-12-31-22-31-38-76_df198e732186825c8df26e3c5a10d7cd.jpg](https://github.com/user-attachments/assets/8bbd2e61-bc27-47b6-85ef-cf246960021b)

I don't know why break this.

Reply:
> > OK.
> 
> So please change it?

Yes